db/migrate/20180727213418_add_index_to_form526_opt_in.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexToForm526OptIn < ActiveRecord::Migration
+  disable_ddl_transaction!
```

Comment:
Out of curiosity, why is this necessary?

Reply:
The short answer is because im adding a concurrent index which needs to be run outside of a transaction... but rails wraps each migration inside of a transaction (by default). So it has to be disabled.

Reply:
What's the advantage of a concurrent index?

Reply:
Its probably overkill for this case but it keeps the table from being locked out (It can still be interacted with). So it allows for zero downtime deployments.

Reply:
I see, cool! I don't think it's overkill since it doesn't add much complexity and it seems like a nice benefit. Thanks for enlightening me :)